### PR TITLE
Changes for move and field makers logic

### PR DESCRIPTION
**Pull request by SachkoMaxim**
Added move count cleaner.
Added zooming to actual size after changing cells numbers in field. 
Added logic to move button.

### CODE REVIEW

**Comment by pavukach** [comment]
Code becomes laggy so I suggest:
```
function updateFieldImages() {
  const cells = document.querySelectorAll('.cell');
  cells.forEach((cell) => {
    const element = cell as HTMLElement;
    const x = parseInt(element.dataset.x || '0');
    const y = parseInt(element.dataset.y || '0');
    const fieldCell = field.getCell(x, y);

    if (fieldCell.picture) {
      let img = element.querySelector('img');
      if (!img) {
        img = document.createElement('img');
        element.appendChild(img);
      }
      img.src = fieldCell.picture;
    }
  });
}
```
and clear the dummycell action. 
Also think about caching cells and not doing querry selector every time.

File: src/index.ts [hunk 1]
```diff
@@ -121,9 +119,41 @@ function handleCellClick(event: MouseEvent) {
...
+    if (fieldCell.picture) {
+      let img = element.querySelector('img');
+      if (!img) {
+        img = document.createElement('img');
+        element.appendChild(img);
+      }
+      img.src = fieldCell.picture;
+      img.alt = `Cell at (${x}, ${y})`;
+      img.style.width = `${element.offsetWidth}px`;
+      img.style.height = `${element.offsetHeight}px`;
```
Comment (pavukach) [inline src/index.ts]: Remove this 3 lines.

File: src/index.ts [hunk 2]
```diff
@@ -121,9 +119,41 @@ function handleCellClick(event: MouseEvent) {
...
+    for (let y = 0; y < fieldSizeY; y++) {
+      const cell = field.getCell(x, y);
+      cell.action(x, y);
+    }
+  }
+  updateFieldImages();
+}
+
+function updateFieldImages() {
+  const cells = document.querySelectorAll('.cell');
```
Comment (pavukach) [inline src/index.ts]: Get all cells in the begginning and store them in array?

**Review by pavukach — CHANGES_REQUESTED** [review]
Wrote above.

File: src/index.ts (hunk 1 above)
Comment (SachkoMaxim) [reply src/index.ts]: Ok

File: src/index.ts (hunk 2 above)
Comment (SachkoMaxim) [reply src/index.ts]: Ok

**Review by pavukach — APPROVED** [review]

**Pull request merged**